eng/Versions.props

```diff
@@ -32,9 +32,9 @@
     <CodeStyleLayerCodeAnalysisVersion>3.6.0-2.final</CodeStyleLayerCodeAnalysisVersion>
     <MicrosoftCodeAnalysisTestingVersion>1.0.1-beta1.20407.3</MicrosoftCodeAnalysisTestingVersion>
     <CodeStyleAnalyzerVersion>3.7.0-3.20271.4</CodeStyleAnalyzerVersion>
-    <VisualStudioEditorPackagesVersion>16.4.248</VisualStudioEditorPackagesVersion>
+    <VisualStudioEditorPackagesVersion>16.8.39</VisualStudioEditorPackagesVersion>
```

Comment:
This might break CI if it's newer than what being shipped in VS

Reply:
> This might break CI if it's newer than what being shipped in VS. 

How would it break CI?  Won't CI just use what's actually on our CI machines?

Reply:
Updating the version along won't, but if we are going to use anything new in the version, then it will.